app/api/dao/user.py

```diff
@@ -209,7 +215,10 @@ def update_user_profile(user_id: int, data: Dict[str, str]):
 
             # username should be unique
             if user_with_same_username:
-                return messages.USER_USES_A_USERNAME_THAT_ALREADY_EXISTS, HTTPStatus.BAD_REQUEST
+                return (
+                    messages.USER_USES_A_USERNAME_THAT_ALREADY_EXISTS,
+                    HTTPStatus.BAD_REQUEST,
+                )
```

Comment:
hmm.... apart from adding utcnow() , i can see that you have done the formatting of most of the returning messages with response then i think you can change the title of the issue or this PR to something relevant to it so, that it can easily help anyone to understand exactly what all you have done in this PR ... [just a suggestion,] BTW, its a greatly needed change in the PR

Reply:
Oopss.. I think this must be the autopep or something I accidentally set to 'ON' in my VSCode IDE. I'll fix it and push again.. Nice catch, @robotjellyzone. Thanks again. 😉

Reply:
mmm. I think it's the Prettier on VSCode doing this auto-prettying stuff. hehehe.. should I just add a little explanation on the PR that I did a little formatting a bit, rather than changing it back to how it was. Will it make any different if I reverse it back? I think the difference is only from one line to two lines. @robotjellyzone, what do you think?

Reply:
I think you should mention this in your description of this PR but I think then you have to go through the various other files as well to make sure this consistency should be maintained with other response messages as well  so, maybe let's just wait for @isabelcosta to suggest !!

Reply:
Alright, I think before I change anything, I'll wait for @isabelcosta to give her thoughts then. 😉

Reply:
I found the culprit. I set both Python > Linting: Lint On Save and Python > Linting: Pylint Enabled too True. Mmm... I actually quite like these options to be on, just to make sure the codes abide the Linting standards. I'll probably mention this is PR as additional note and if need to, for the rest of the files we can schedule some code reformatting period (once a month or every 2-3 month?) just to make sure code quality is kept in check. It'll be similar to what I did with [PR#478](https://github.com/anitab-org/mentorship-backend/pull/478) last time, but in smaller scale as opposed to the whole project (probably folder by folder). Anyway, we'll wait for @isabelcosta, or perhaps @ramitsawhney27 can also give us advise on this 😆 .

Reply:
> code reformatting period (once a month or every 2-3 month?)

This is interesting definitely! However, the real solution I think would be, doing a refactor on the style, and ensure that from a point in time, contributors are required to run a linting tool that does such code formatting. Just like we started talking for `black` tool. I am just not sure about which tool to use 🤔